src/map/script.cpp

```diff
@@ -11647,40 +11647,38 @@ BUILDIN_FUNC(getareadropitem)
  *------------------------------------------*/
 BUILDIN_FUNC(enablenpc)
 {
-	const char *str = script_getstr(st,2);
-	if (npc_enable(str,1))
-		return SCRIPT_CMD_SUCCESS;
-
-	return SCRIPT_CMD_FAILURE;
-}
+	npc_data* nd = nullptr;
```

Comment:
```suggestion
	npc_data* nd;
```
Doesn't need an assignment since we don't do any pre-checks and it gets it's assignment a few lines below.